starter-code/bin/seeds.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const Celebrity = require("../models/celebrity.model");
+
+const DB_NAME = "starter-code";
+
+mongoose.connect(`mongodb://localhost/${DB_NAME}`, {
+  useCreateIndex: true,
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+const celebrities = [
+  { name: "Donal Trump", occupation: "Golfer", catchPhrase: "Fake News!" },
```

Comment:
try to keep the same format as the objects below. Either all in the same line or in different lines.